core/templates/services/math-interactions.service.ts

```diff
@@ -162,13 +171,13 @@ export class MathInteractionsService {
         'Your answer contains an invalid term: ' + invalidIntegers[0]);
       return false;
     }
-    let invalidMultiTerms = expressionString.match(
-      /([a-zA-Z]\d)/g);
+    let invalidMultiTerms = expressionString.match(/([a-zA-Zα-ωΑ-Ω]\d+)/g);
```

Comment:
Just to check -- I thought in our parser, Greek letters were spelled out, like "alpha"? If so, should test that case, I think... (presumably we have a regex for "all greek letters" somewhere).

Reply:
Yes, you're right. I've updated the condition and tests cases accordingly, thanks!